sklearn/linear_model/tests/test_ridge.py

```diff
@@ -1008,26 +1008,53 @@ def test_n_iter():
         assert reg.n_iter_ is None
 
 
-def test_ridge_fit_intercept_sparse():
-    X, y = make_regression(n_samples=1000, n_features=2, n_informative=2,
-                           bias=10., random_state=42)
+@pytest.mark.parametrize('solver', ['sparse_cg', 'auto'])
+def test_ridge_fit_intercept_sparse(solver):
+    X, y = _make_sparse_offset_regression(n_features=20, random_state=0)
+    X_csr = sp.csr_matrix(X)
+
+    # for now only sparse_cg can fit an intercept with sparse X with default
+    # tol and max_iter, sag should raise a warning and is handled in
+    # test_ridge_fit_intercept_sparse_sag
+    # "auto" should switch to "sparse_cg"
```

Comment:
it this comment still relevant? I don't sag warnings caught here.
If you update the comment can you write what you answered to me just below? thanks

Reply:
thanks! updated the comment. you don't see warnings here because sag's behaviour in this configuration is tested separately in `test_ridge_fit_intercept_sparse_sag`